src/senselab/audio/tasks/speaker_embeddings/api.py

```diff
@@ -37,7 +37,7 @@ def extract_speaker_embeddings_from_audios(
         >>> print(embeddings[0].shape)
         torch.Size([192])
     """
-    if isinstance(model, HFModel):  # TODO: check that this is a speechbrain model!!!
```

Comment:
seems you were too excited

Reply:
haha do you have any idea on how we can check if a local model is a `speechbrain` model?

Reply:
I think we can trust the SpeechBrain functionality to check that for us. This is the function that ends of getting called when you give the source which handles if it's HuggingFace or a local file
https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/utils/fetching.py
As well as this which seems to imply that they'll handle non-speech brain files
https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/inference/classifiers.py#L30

Reply:
will think about that, thanks!